src/Asset/EntrypointLookup.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\WebpackEncoreBundle\Asset;
+
+/**
+ * Returns the CSS or JavaScript files needed for a Webpack entry.
+ *
+ * This reads a JSON file with the format of Webpack Encore's entrypoints.json file.
+ * @final
+ */
+class EntrypointLookup
+{
+    private $entrypointJsonPath;
+
+    private $entriesData;
+
+    private $returnedFiles = [];
```

Comment:
`array()`

Reply:
Or, we might be okey with `[]` since fabbot is not active here.

Reply:
Yea, not sure. We use `[]` on MakerBundle, but were forced to not use fabbot due to this :/. Using fabbot is ideal.

Reply:
no reason to use array() here, as there is merge conflict to save us from